editor/components/block-mover/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import classnames from 'classnames';
+import { first, last } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import IconButton from 'components/icon-button';
+
+function BlockMover( { onMoveUp, onMoveDown, isFirst, isLast } ) {
+	return (
+		<div className="editor-block-mover">
+			<IconButton
+				className={ classnames( 'editor-block-mover__control', { 'is-disabled': isFirst } ) }
```

Comment:
Maybe we should keep it this way in this PR and drop the `editor` prefix from all the components in a specific PR. (To avoid having a mix of conventions)